src/ui/src/session.cpp

```diff
@@ -189,8 +190,11 @@ WindowSession::restore( const std::function<ViewInterface*()>& view_factory,
     const auto& session = SessionInfo::getSynced();
 
     std::vector<SessionInfo::OpenFile> session_files = session.openFiles( windowId_ );
-    LOG_DEBUG << "Session returned " << session_files.size();
+    const size_t count = session_files.size();
```

Comment:
Another settings place. Let's remove reserve

Reply:
fix it https://github.com/variar/klogg/commit/a4f9cad9eceff87a667f1c6cec939f383153a6e3